File: BaoLinngQiu/Assets/Script001/moves.js
var flag : boolean ;
function Update () 
{
	if(this.transform.position.y>6.5)
	{
		flag=true;
	}else if(this.transform.position.y<6)
	{
		flag=false;
	}
	
	if(flag)
	{
		this.transform.position.y-=0.12;
	}else 
	{
		this.transform.position.y+=0.12;
	}
	
}

File: BaoLinngQiu/Assets/Scripts/Moves.cs
using UnityEngine;
using System.Collections;

public class Moves : MonoBehaviour {
	public bool flag;
	private float y = 0;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
		if(this.transform.position.y>6.5f)
		{
			flag=true;
		}else if(this.transform.position.y<6)
		{
			flag=false;
		}
		
		if(flag)
		{
			y-=0.12f;
			this.transform.position=
				new Vector3(this.transform.position.x,this.transform.position.y+y,this.transform.position.z);
		}else 
		{
			y+=0.12f;
			this.transform.position=
				new Vector3(this.transform.position.x,this.transform.position.y+y,this.transform.position.z);
		}
	}
}
